<launch>
  <arg name="rviz" default="false"/>

  <node pkg="hba" type="loopClosure_offline" name="hba" output="screen">

    <param name="if_visualize" type="bool" value="$(arg rviz)"/>
    <param name="stack_size" type="int" value="50"/>  <!-- stack lidar frame for better gicp -->
    <param name="leaf_size" type="double" value="0.1"/>  <!-- stack lidar frame for better gicp -->
    <param name="data_path" type="string" value="/home/jack/FastLab/Multi-UAV-Map-Post-Process/data/body_pcd/"/>
    <!-- <param name="data_path" type="string" value="/home/ckx/workbench/BA/train2/"/> -->

  </node>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz"
    args="-d $(find hba)/rviz_cfg/loopClosure.rviz" />
  </group>
</launch>